rest_auth_toolkit/serializers.py

```diff
@@ -24,6 +26,23 @@ class Meta:
             'password': {'style': {'input_type': 'password'}},
         }
 
+    def validate(self, data):
+        user = User(**data)
+
+        password = data.get('password')
+
+        errors = dict()
+        try:
+            validators.validate_password(password=password, user=user)
+
```

Comment:
Style note: blank lines should separate meaningful blocks (new function, if block, etc.), this one increases vertical space without being useful.